docs/source/usage.rst

```diff
@@ -6,18 +6,27 @@ libMobility offers a set of solvers to compute the hydrodynamic displacements of
 In particular, libMobility solvers can compute the different elements on the right hand side of the following stochastic differential equation:
 
 .. math::
-   
-   d\boldsymbol{X} = \boldsymbol{\mathcal{M}}\boldsymbol{F}dt + \text{prefactor}\sqrt{2 k_B T\boldsymbol{\mathcal{M}}}d\boldsymbol{W}
 
-.. hint:: See the :ref:`solvers` section for a list of available solvers.
+   \begin{bmatrix}\boldsymbol{dX}\\\boldsymbol{d\tau}\end{bmatrix} = \boldsymbol{\mathcal{\Omega}}\begin{bmatrix}\boldsymbol{F}\\\boldsymbol{T}\end{bmatrix}dt + \text{prefactor}\sqrt{2 k_B T \boldsymbol{\mathcal{\Omega}}}d\boldsymbol{W}
+
+Where:
 
-Where :math:`d\boldsymbol{X}` are the linear displacements, prefactor is a user-provided prefactor and :math:`d\boldsymbol{W}` is a collection of i.i.d Weinner processes and T is the temperature. Finally :math:`\boldsymbol{M}` represents the mobility tensor. 
+- :math:`d\boldsymbol{X}` are the linear displacements
+- :math:`\boldsymbol{d\tau}` are the angular displacements
```

Comment:
Is there a typo here? dX has no bold 'd' but dTau has a bold 'd'. Not sure if it's a notation I'm just unfamiliar with

Reply:
Just a typo, thanks for catching it!